### DISCUSSION TITLE
I have forgot the login credentials for my localhost server using Payload. Anyone know how to fix this

### DISCUSSION THREAD

**Discussion by tanbiralam** [payloadcms/payload#5437] [General]
![image](https://github.com/payloadcms/payload/assets/57638199/397eefe3-198c-4aa2-a50e-8d6dd51e3015)
I am building a web app using payload cms, but when I try to log in through the local dev server, it gives the same error that the email or password is incorrect. I have reinstalled the packages, but it still can't solved yet.

**Comment by andershermansen** [comment]
The easiest is to connect to the database and remove all users. Then when you load payload again it will ask you to create user and password.

**Reply by Kingbonwa1994** [reply]
thanks, this works
